src/components/Tabset/utils.js

```diff
@@ -44,6 +44,13 @@ export function getChildrenTotalWidthUpToClickedTab(children, index) {
 }
 
 export function isNotSameChildren(children, prevChildren) {
+    if (!Array.isArray(children)) {
```

Comment:
I will avoid negating at the beginning of a function. Why? because negations in an `if` do not read well.

try this
```js
if (Array.isArray(children)) {
    return children.some ...
}
return children.props.name === prevChildren.props.name;
```
simple :) 

also, try to write some tests for you fix

Reply:
thanks @maxxgreene . I'm gonna update it